go/adapter.go

```diff
@@ -2,18 +2,23 @@ package observe
 
 import (
 	"context"
+	"time"
 
 	"github.com/tetratelabs/wazero"
 )
 
+type AdapterConfig struct {
+	FlushPeriod int
```

Comment:
this may be better as `time.Duration` vs. `int`

Reply:
I thought i changed this. Looks like maybe this is unused.